resources/public/api/conf/2.0/examples/adjustments/foreign_property_accounting_adjustments_submit_request.json

```diff
@@ -0,0 +1,36 @@
+{
+  "foreignProperty": {
+    "income": {
+      "rentIncome": 123.12,
+      "premiumsOfLeaseGrant": 123.12,
+      "foreignTaxTakenOff": 123.12,
+      "otherPropertyIncome": 123.12
+    },
+    "expenses": {
+      "premisesRunningCosts": 123.12,
+      "repairsAndMaintenance": 123.12,
+      "financialCosts": 123.12,
+      "professionalFees": 123.12,
+      "travelCosts": 123.12,
+      "costOfServices": 123.12,
+      "residentialFinancialCost": 123.12,
+      "other": 123.12,
+      "consolidatedExpenses": 123.12
+    }
+  },
+  "foreignFhlEea": {
+    "income": {
+      "rentIncome": 123.12
+    },
+    "expenses": {
+      "premisesRunningCosts": 123.12,
+      "repairsAndMaintenance": 123.12,
+      "financialCosts": 123.12,
+      "professionalFees": 123.12,
+      "costOfServices": 123.12,
+      "travelCosts": 123.12,
+      "other": 123.12,
+      "consolidatedExpenses": 123.12
+    }
+  }
+}
```

Comment:
The example given wouldn't be allowed to be done as a request.

You should give examples of both a regular and consolidated request.

Also you aren't able to submit both a foreignProperty object and a foreignFhlEea object in the same request.